prow/gerrit/adapter/adapter.go

```diff
@@ -376,7 +381,15 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 	case client.Merged:
 		postsubmits, err := cache.GetPostsubmits(trimmedHostPath, func() (string, error) { return baseSHA, nil }, func() (string, error) { return change.CurrentRevision, nil })
 		if err != nil {
-			return fmt.Errorf("failed to get inRepoConfig for Postsubmits: %w", err)
+			attempts := 1
+			// Gerrit InRepoConfig can flake. Try again just in case.
+			for attempts < inRepoConfigRetries && err != nil {
```

Comment:
Do you need an `attempts++` somewhere? (same below)

Reply:
oof yea I do. Done